packages/design-tokens/CHANGELOG.md

```diff
@@ -7,19 +7,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.0]
+
+### Added
+
+- **BREAKING:** Following the unintentional breaking change in `4.2.0` we are now exporting the types from root index instead of requiring deep imports ([#340](https://github.com/MetaMask/metamask-design-system/pull/340)). See the [migration guide](./MIGRATION.md#from-version-410-to-500) for details.
+
+- Added 8 new colors (4 muted-hover & 4 muted-pressed) to design-tokens Figma Json. ([#325](https://github.com/MetaMask/metamask-design-system/pull/325))
+
```

Comment:
Updating changelog to note breaking change according to [Guide to Changelogs](https://github.com/MetaMask/contributor-docs/blob/add-changelogs-guidelines/docs/changelogs.md) as well as linking to the migration guide